.pylintrc

```diff
@@ -48,7 +48,7 @@ unsafe-load-any-extension=no
 # A comma-separated list of package or module names from where C extensions may
 # be loaded. Extensions are loading into the active Python interpreter and may
 # run arbitrary code
-extension-pkg-whitelist=
+extension-pkg-allowlist=
```

Comment:
This instance of the term whitelist is part of pylint, not part of our code, so we should revert the change in this line.